nixpkgs-review looks good 👍️


## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 548804`
Commit: `28976779db411234724f5ef7b465adcb6662f51b`

---
### `x86_64-linux`
<details>
  <summary>:x: 3 packages failed to build:</summary>
  <ul>
    <li>auto-editor</li>
    <li>minitube</li>
    <li>shira</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 132 packages built:</summary>
  <ul>
    <li>adl</li>
    <li>ani-cli</li>
    <li>bluebubbles</li>
    <li>castero</li>
    <li>catt</li>
    <li>celluloid</li>
    <li>cine</li>
    <li>clapper</li>
    <li>clapper-enhancers</li>
    <li>cliamp</li>
    <li>cplay-ng</li>
    <li>curseradio</li>
    <li>delfin</li>
    <li>deno</li>
    <li>dmlive</li>
    <li>dra-cla</li>
    <li>era</li>
    <li>ff2mpv</li>
    <li>ff2mpv-go</li>
    <li>gallery-dl</li>
    <li>gleam</li>
    <li>gonic</li>
    <li>gophertube</li>
    <li>gpodder</li>
    <li>gtk-pipe-viewer</li>
    <li>haruna</li>
    <li>hydrus</li>
    <li>hypnotix</li>
    <li>invidtui</li>
    <li>jellyfin-mpv-shim</li>
    <li>jellyfin-tui</li>
    <li>jellytui</li>
    <li>jftui</li>
    <li>kdePackages.audiotube</li>
    <li>kdePackages.kalarm</li>
    <li>kdePackages.plasmatube</li>
    <li>kikoplay</li>
    <li>klipperscreen</li>
    <li>kodiPackages.sendtokodi</li>
    <li>kopuz</li>
    <li>linux-wallpaperengine</li>
    <li>liquidsoap</li>
    <li>lollypop</li>
    <li>losange</li>
    <li>mealie</li>
    <li>memento</li>
    <li>mkchromecast</li>
    <li>monophony</li>
    <li>moodle-dl</li>
    <li>mopidy-youtube (mopidyPackages.mopidy-youtube)</li>
    <li>mov-cli</li>
    <li>mpc-qt</li>
    <li>mpv</li>
    <li>mpv-handler</li>
    <li>mpvScripts.mpv-playlistmanager</li>
    <li>mpvpaper</li>
    <li>music-assistant</li>
    <li>parabolic</li>
    <li>parrot</li>
    <li>persepolis</li>
    <li>photoqt</li>
    <li>pinchflat</li>
    <li>pipe-viewer</li>
    <li>pipeline</li>
    <li>plex-mpv-shim</li>
    <li>previewqt</li>
    <li>python313Packages.beets-ytimport</li>
    <li>python313Packages.bgutil-ytdlp-pot-provider</li>
    <li>python313Packages.mpv</li>
    <li>python313Packages.tubeup</li>
    <li>python313Packages.yark</li>
    <li>python313Packages.yt-dlp</li>
    <li>python314Packages.beets-ytimport</li>
    <li>python314Packages.bgutil-ytdlp-pot-provider</li>
    <li>python314Packages.mpv</li>
    <li>python314Packages.tubeup</li>
    <li>python314Packages.yark</li>
    <li>python314Packages.yt-dlp</li>
    <li>qimgv</li>
    <li>qmplay2 (qmplay2-qt6)</li>
    <li>qmplay2-qt5</li>
    <li>qtgreet</li>
    <li>quarto</li>
    <li>quartoMinimal</li>
    <li>radiance-vj</li>
    <li>radio-cli</li>
    <li>radioboat</li>
    <li>rstudio</li>
    <li>rstudio-server</li>
    <li>rstudioServerWrapper</li>
    <li>rstudioWrapper</li>
    <li>scdl</li>
    <li>sharedown</li>
    <li>simple-live-app</li>
    <li>somafm-cli</li>
    <li>spotdl</li>
    <li>stremio-linux-shell</li>
    <li>subtitleedit</li>
    <li>svp</li>
    <li>switchfin</li>
    <li>szurubooru.server</li>
    <li>tartube-yt-dlp</li>
    <li>tests.home-assistant-components.media_extractor</li>
    <li>tomato-c</li>
    <li>tuba</li>
    <li>varia</li>
    <li>vidcutter</li>
    <li>video-downloader</li>
    <li>vimPlugins.ddc-filter-matcher_head</li>
    <li>vimPlugins.ddc-filter-sorter_rank</li>
    <li>vimPlugins.ddc-fuzzy</li>
    <li>vimPlugins.ddc-source-around</li>
    <li>vimPlugins.ddc-source-file</li>
    <li>vimPlugins.ddc-source-lsp</li>
    <li>vimPlugins.ddc-ui-native</li>
    <li>vimPlugins.ddc-ui-pum</li>
    <li>vimPlugins.ddc-vim</li>
    <li>vimPlugins.denops-vim</li>
    <li>vimPlugins.peek-nvim</li>
    <li>vimPlugins.vim-pluto</li>
    <li>vrcvideocacher</li>
    <li>windmill</li>
    <li>wtwitch</li>
    <li>yewtube</li>
    <li>youtube-tui</li>
    <li>yt-dlp</li>
    <li>ytcc</li>
    <li>ytdl-sub</li>
    <li>ytdownloader</li>
    <li>ytfzf</li>
    <li>ytmdl</li>
    <li>ytui-music</li>
  </ul>
</details>


---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>auto-editor</summary>
<pre>/build/nimcache/@mav.nim.c: In function &#x27;initEnCtx__ni_u65&#x27;:
/build/nimcache/@mav.nim.c:1428:36: error: &#x27;AVCodec&#x27; has no member named &#x27;sample_fmts&#x27;
 1428 |                 if (!!(((*codec_p0).sample_fmts == ((enum AVSampleFormat*) NIM_NIL)))) goto LA7_;
      |                                    ^
/build/nimcache/@mav.nim.c:1429:57: error: &#x27;AVCodec&#x27; has no member named &#x27;sample_fmts&#x27;
 1429 |                 (*encoderCtx_1).sample_fmt = (*codec_p0).sample_fmts[((NI)0)];
      |                                                         ^
/build/nimcache/@mav.nim.c: In function &#x27;addStream__ni_u3080&#x27;:
/build/nimcache/@mav.nim.c:1844:49: error: &#x27;AVCodec&#x27; has no member named &#x27;sample_fmts&#x27;
 1844 |                 (*ctx_1).sample_fmt = (*codec_1).sample_fmts[((NI)0)];
      |                                                 ^
CC: nimcrypto/utils.nim
CC: nimcrypto/hash.nim
CC: nimcrypto/sha.nim
CC: cache.nim
CC: resampler.nim
Error: execution of an external compiler program &#x27;gcc -c -w -fmax-errors=3 -fno-strict-aliasing -pthread -msse4.2 -I./build/include -fno-signaling-nans -fno-math-errno -fno-trapping-math -fno-signed-zeros -Wno-incompatible-pointer-types -Wno-stringop-overflow -O3 -fno-ident -fno-math-errno   -I/nix/store/6gy6k7wbhrswdzy7kw0cwnf00lljnjqq-nim-unwrapped-2.2.10/nim/lib -I/build/source/src -o /build/nimcache/@mav.nim.c.o /build/nimcache/@mav.nim.c&#x27; failed with exit code: 1


build phase failed</pre>
</details>
<details>
<summary>minitube</summary>
<pre>      |                                                                                         ^~~~~~~~~~~~~~
lib/qt-reusable-widgets/iconutils.h: In static member function &#x27;static void IconUtils::setIcon(T*, const char*)&#x27;:
lib/qt-reusable-widgets/iconutils.h:23:46: warning: &#x27;void QGuiApplication::paletteChanged(const QPalette&amp;)&#x27; is deprecated: Handle QEvent::ApplicationPaletteChange instead [8;;https://gcc.gnu.org/onlinedocs/gcc-15.3.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
   23 |         obj-&gt;connect(qApp, &amp;QGuiApplication::paletteChanged, obj, setObjIcon);
      |                                              ^~~~~~~~~~~~~~
/nix/store/n3f02b0nsrjsmpcc44dwcky8prgqs97q-qtbase-6.11.1/include/QtGui/qguiapplication.h:161:89: note: declared here
  161 |     QT_DEPRECATED_VERSION_X_6_0(&quot;Handle QEvent::ApplicationPaletteChange instead&quot;) void paletteChanged(const QPalette &amp;pal);
      |                                                                                         ^~~~~~~~~~~~~~
src/standardfeedsview.cpp: In member function &#x27;virtual void StandardFeedsView::showEvent(QShowEvent*)&#x27;:
src/standardfeedsview.cpp:174:47: warning: unused parameter &#x27;event&#x27; [8;;https://gcc.gnu.org/onlinedocs/gcc-15.3.0/gcc/Warning-Options.html#index-Wno-unused-parameter-Wunused-parameter8;;]
  174 | void StandardFeedsView::showEvent(QShowEvent *event) {
      |                                   ~~~~~~~~~~~~^~~~~
src/standardfeedsview.cpp: In member function &#x27;virtual void StandardFeedsView::hideEvent(QHideEvent*)&#x27;:
src/standardfeedsview.cpp:184:47: warning: unused parameter &#x27;event&#x27; [8;;https://gcc.gnu.org/onlinedocs/gcc-15.3.0/gcc/Warning-Options.html#index-Wno-unused-parameter-Wunused-parameter8;;]
  184 | void StandardFeedsView::hideEvent(QHideEvent *event) {
      |                                   ~~~~~~~~~~~~^~~~~
src/playlistmodel.cpp: In member function &#x27;virtual bool PlaylistModel::removeRows(int, int, const QModelIndex&amp;)&#x27;:
src/playlistmodel.cpp:311:16: warning: unused variable &#x27;video&#x27; [8;;https://gcc.gnu.org/onlinedocs/gcc-15.3.0/gcc/Warning-Options.html#index-Wno-unused-variable-Wunused-variable8;;]
  311 |         Video *video = videos.takeAt(position);
      |                ^~~~~</pre>
</details>
<details>
<summary>shira</summary>
<pre>no configure script, doing nothing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built shiradl-1.8.5-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/shiradl-1.8.5...OK
Repacking wheel as ./shiradl-1.8.5-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;pythonRuntimeDepsCheckHook&quot; }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for shiradl-1.8.5-py3-none-any.whl
  - ytmusicapi==1.12.1 not satisfied by version 1.12.2</pre>
</details>
</details>

--- a/pkgs/by-name/de/deno/package.nix
+++ b/pkgs/by-name/de/deno/package.nix
@@ -43,6 +43,7 @@ rustPlatform.buildRustPackage (finalAttrs: {
   outputs = [
     "out"
     "denort"
+    "libdenort"
   ];
 
   src = fetchFromGitHub {
@@ -244,6 +245,7 @@ rustPlatform.buildRustPackage (finalAttrs: {
 
   postInstall = ''
     moveToOutput "bin/denort" "$denort"
+    moveToOutput "lib/libdenort${stdenv.hostPlatform.extensions.sharedLibrary}" "$libdenort"
 
     # Remove non-essential binaries like test_server
     find $out/bin/* -not -name "deno" -delete

